src/components/pages/MyProfile.js

```diff
@@ -0,0 +1,40 @@
+import { useSelector } from 'react-redux';
+import { selectMission } from '../../redux/Mission/MissionSice';
+
+const MyProfile = () => {
+  // Make sure you're using the correct slice name here
+  const { rockets } = useSelector((state) => state.rockets);
+  const { missionData, joinedMissions } = useSelector(selectMission);
+  const missionID = joinedMissions;
+
+  const Missioners = missionData.filter((mission) => missionID.includes(mission.mission_id));
+
+  return (
+    <>
+      <section className="myProfileContainer">
+        <div className="sectionA">
+          <h2>My Missions</h2>
+          <ul className="rockets">
+            {Missioners.map((mission) => (
+              <li key={mission.mission_id} className="missioName">
+                {mission.mission_name}
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div className="sectionB">
+          <h2>My Rockets</h2>
+          <ul className="rockets">
+            {rockets
+              .filter((rocket) => rocket.reserved)
+              .map((rocket) => (
+                <li key={rocket.id}>{rocket.rocket_name}</li>
+              ))}
+          </ul>
+        </div>
+      </section>
+    </>
```

Comment:
- [ ] For the profile page , the same thing will also be noticed , Kindly ensure the state are saved when the component is re-rendered , when navigating through component 👍 

### Before Re-rendered 

![Capture](https://github.com/GhostEsso/SpaceTravelersHub/assets/80458199/7d8a6e83-2be6-4102-a48a-3cf9eee6adf7)

### After Re-rendered 

![Capture1](https://github.com/GhostEsso/SpaceTravelersHub/assets/80458199/955bc557-81da-4ea5-80ba-1a501830ed4a)